apps/gro/fields/address.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 module.exports = {
-  address: {
-    className: ['address']
+  'address-textarea': {
+    validate: ['required'],
+    'ignore-defaults': true,
+    formatter: ['trim', 'hyphens']
```

Comment:
if one specifies a list of formatters, is it necessary to also ignore the others?